lib/experimental/fonts/font-library/class-wp-rest-font-families-controller.php

```diff
@@ -254,15 +198,7 @@ public function delete_item( $request ) {
 			);
 		}
 
-		$deleted = parent::delete_item( $request );
-
-		if ( is_wp_error( $deleted ) ) {
-			return $deleted;
-		}
-
-		foreach ( $this->get_font_face_ids( $font_family_id ) as $font_face_id ) {
-			wp_delete_post( $font_face_id, true );
-		}
```

Comment:
Deleting child font families is now handled by callbacks hooked into `deleted_post`, so we don't need to do it here in the controller